### PR TITLE
Deploy App Using React Scripts

### LINKED ISSUE #94: Deploy App
The app needs to be published using S3. The biggest change here will be getting the deployment script situated in the `package.json`

### PR DESCRIPTION
**Pull request by wgranger**
I'm utilizing `react-scripts` here to handling building the app, then appending the correct path `alice.zooniverse.org` for deployment and `preview.zooniverse.org/alice` (alice.preview.zooniverse.org) for staging to send the site to the S3 bucket with `publisssh`. 

I've also created a `config.js` file to handle URLs that must dynamically change depending on the environment.

Closes #94

### CODE REVIEW

**Comment by wgranger** [comment]
This can be tested with the dry run flag `-d` https://www.npmjs.com/package/publisssh

**Comment by wgranger** [comment]
This should be working, but I had to hack the `panoptes-client` in the `node_modules` to get [this fix](https://github.com/zooniverse/panoptes-javascript-client/pull/128) in

**Comment by wgranger** [comment]
Also removed the `@zooniverse/panoptes-js` dependency so I won't have to do a fix similar to the PJC one on that package. It was only being used to fetch subjects.

**Comment by wgranger** [comment]
This should be working with the newly updates panoptes-client

File: package.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
     "@fortawesome/react-fontawesome": "^0.1.4",
     "@storybook/addon-storyshots": "^5.2.5",
     "@zooniverse/grommet-theme": "^2.1.0",
-    "@zooniverse/panoptes-js": "^0.1.1",
```
Comment (mcbouslog) [inline package.json]: I'm not super familiar with `panoptes-js`, was there anything unique it was doing, that PJC doesn't do?

**Comment by mcbouslog** [comment]
Oh can be in subsequent PR - add urls (staging and production) to README

File: package.json (hunk 1 above)
Comment (mcbouslog) [reply package.json]: Related updated requests look good, nevermind.

**Pull request merged**